k8s/serviceinstance/loop.go

```diff
@@ -111,24 +114,43 @@ func handleAddServiceInstance(
 		InstanceID:        serviceInstance.Spec.ID,
 		ServiceID:         sc.ID,
 		PlanID:            serviceInstance.Spec.PlanID,
-		AcceptsIncomplete: false,
+		AcceptsIncomplete: true,
 		Parameters:        serviceInstance.Spec.Parameters,
 	}
 	finalServiceInstanceState := data.ServiceInstanceStateFailed
+	finalServiceInstanceStateReason := ""
 	defer func() {
```

Comment:
I actually handled this differently. I was doing an update _before_ beginning polling and not hanging onto the `*data.ServiceInstance` that was being returned from that, so the one I previously had was stale when it came time to update after polling was done. Now I'm grabbing and hanging onto that updated `*data.ServiceInstance` so the next update doesn't fail.